openmp/tools/archer/tests/races/DRB006-indirectaccess2-orig-yes.c

```diff
@@ -0,0 +1,161 @@
+/*
+ * DRB006-indirectaccess2-orig-yes.c -- Archer testcase
+ */
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+//
+// See tools/archer/LICENSE.txt for details.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+/*
+Copyright (c) 2017, Lawrence Livermore National Security, LLC.
```

Comment:
@jdoerfert what is our policy regarding test case copyright?

Reply:
AFAIK, tests, like everything else in llvm, need to be licensed like the rest, so Apache-2.0 WITH LLVM-exception.
This file is super weird.
`See tools/archer/LICENSE.txt for details.`
Is not even a file, and probably should not be there to begin with.

I am unsure if we can even take this file with the copyright in it. We can ask the board, or avoid the file all together.